src/hooks/queries/useAuthService.ts

```diff
@@ -0,0 +1,48 @@
+import { useMutation } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+import { LoginResponse } from '@/models/auth/authApiModels';
+import { useAuthStore } from '@/stores/authStore';
+import { useModalStore } from '@/stores/modalStore';
+import type { LoginForm } from '@/models/auth/authModels';
+import { login } from '../../services/api/authApi';
+import { useUserStore } from '@/stores/userStore';
+import { userData } from '@/services/api/userApi';
+
+export const useLogin = () => {
+  const { closeModal } = useModalStore();
+  const setUser = useUserStore((state) => state.setUser);
+
+  return useMutation<LoginResponse, AxiosError, LoginForm>({
+    mutationFn: login,
+    onSuccess: async (response) => {
+      const { accessToken, refreshToken } = response.data;
+
+      // 토큰을 전역 저장소에 저장 (zustand에서 persist로 localStorage에 저장)
+      useAuthStore.getState().login(accessToken, refreshToken);
```

Comment:
위에 `const loginAuthStore = useAuthStore((state) => state.login);` 추가하고 여기를 다음과 같이 바꾸면 `setUser`와 통일성이 있을 것 같아요.

```suggestion
      loginAuthStore(accessToken, refreshToken);
```

Reply:
아 여기서는 그래도 되겠네요 axios.ts에서는 직접 접근하여 사용해야해서 작업하다보니 헷갈린 것 같습니다
수정하겠습니다

Reply:
✅ be677dccef09776d5a85f0a6c5075f3856bd58ec